qiskit_optimization/applications/__init__.py

```diff
@@ -74,4 +75,5 @@
     "Tsp",
     "VehicleRouting",
     "VertexCover",
+    "SKModel"
```

Comment:
It also needs adding above to the other list that will form the docs. If you keep it in alphabetic order there, which will help the reader find it easier, then it will become an entry between current lines 43 & 44.

Reply:
Thank you. Done.